java-chassis-dependencies/default/pom.xml

```diff
@@ -344,6 +349,11 @@
         <artifactId>hibernate-validator-annotation-processor</artifactId>
         <version>6.0.2.Final</version>
       </dependency>
+      <dependency>
+        <groupId>org.hibernate.validator</groupId>
```

Comment:
Shall we delete groupId>org.hibernate</groupId> above?

Reply:
if delete the old one, maybe will cause some problem for old project